### ISSUE TITLE
internal placeholder :img: is not working

### ISSUE THREAD

**Issue by lpxoainse** [PluginBugs/Issues-ItemsAdder#3955]
### Terms

- [X] I'm using the very latest version of ItemsAdder and its dependencies.
- [X] I am sure this is a bug and it is not caused by a misconfiguration or by another plugin.
- [X] I've looked for already existing issues on the [Issue Tracker](https://github.com/PluginBugs/Issues-ItemsAdder/issues) and haven't found any.
- [X] I already searched on the [plugin wiki](https://itemsadder.devs.beer/) to know if a solution is already known.
- [X] I already searched on the [forums](https://forum.devs.beer/) to check if anyone already has a solution for this.
- [X] I tested this with just ItemsAdder and its dependencies and with a vanilla client of the same version as the Server.

### What happened?

image font in lore is not working, but when i type it to the chat e.g :tool: is working just fine, /papi parse me :tool: OR /papi parse me %img_tool% also works just fine

### Steps to reproduce the issue

1. add custom image font or emoji to item lore
2. /iazip
3. get the item ingame

### Server version

This server is running Paper version 1.21.1-52-master@e08e667 (2024-08-26T18:02:06Z) (Implementing API version 1.21.1-R0.1-SNAPSHOT)

### ItemsAdder Version

ItemsAdder version 4.0.2-alpha-test-7

### ProtocolLib Version

ProtocolLib version 5.3.0-SNAPSHOT-726

### LoneLibs Version

LoneLibs version 1.0.57

### Full Server Log

no errors

### Problematic items yml configuration file (optional)

```yaml
*axe_enchanted_oak.yml
info:
  namespace: ctools
items:
  enchanted_oak_axe:
    enabled: true
    display_name: <light_purple><u>Enchanted Oak Axe</u></light_purple>
    lore:
      - ":tool:"
      - ""
      - <dark_gray>An enchanted oak axe</dark_gray>
    resource:
      generate: true
      material: WOODEN_AXE
      textures:
        - axe/enchanted_oak_axe.png
    durability:
      max_durability: 32
      disappear_when_broken: false
    item_flags:
      - HIDE_ATTRIBUTES
      - HIDE_DESTROYS
      - HIDE_ENCHANTS
      - HIDE_PLACED_ON
      - HIDE_POTION_EFFECTS
      - HIDE_UNBREAKABLE
    drop:
      show_name: true
    events:
      block_break:
        decrement_durability:
          amount: 1


*font_images.yml
info:
  namespace: cfonts
font_images:
  tool:
    permission: tool_img
    show_in_gui: false
    suggest_in_command: false
    path: font/fontimage/tool_img
    scale_ratio: 9
    y_position: 8
```

### Other files, you can drag and drop them here to upload. (optional)

ctools dir
```
+---configs
|   +---axe
|   |       axe_enchanted_oak.yml
|   |       
|   \---shears
|           shears.yml
|           
+---models
|   \---item
\---textures
    +---axe
    |       enchanted_oak_axe.png
    |       
    \---misc
            shears.png
            `

cfonts dir
`+---configs
|       font_images.yml
|       
\---textures
    \---font
        |   
        \---fontimage
                tool_img.png
```

### Screenshots/Videos (you can drag and drop files or paste links)

![image](https://github.com/user-attachments/assets/efae8bff-3dff-4653-bbc1-d4d2e6e4acf3)

**Comment by Andre601** [comment]
Please share your `latest.log` through https://mclo.gs as it help us getting all the necessary info required to solving this issue.

Also, test it with just ItemsAdder and its dependencies, as this could be a plugin preventing ItemsAdder updating the item lore.

**Comment by lpxoainse** [comment]
> test it with just ItemsAdder and its dependencies

I did, still doesn't show
![image](https://github.com/user-attachments/assets/ffa03eb2-a9ca-4d65-9feb-f63ba0014b07)

**Comment by PSYCHEER** [comment]
I'm not seeing whole server log as stated above your answer, provide us this information trough provided pasting service so we can at least try to reproduce your issue, without reproducing it's hard to tell if is it issUe or not

**Comment by lpxoainse** [comment]
> whole server log

https://mclo.gs/5vbk25B
![image](https://github.com/user-attachments/assets/6c91cf6b-cc6a-45d5-9b2d-f87039df338b)

**Comment by LoneDev6** [comment]
Probably fixed by #4179 as it works fine in the latest beta version 4.0.5 (access it on Discord).
